internal/pkg/cli/svc_deploy.go

```diff
@@ -532,6 +542,20 @@ func validateAppVersion(alias string, app *config.Application, appVersionGetter
 	return nil
 }
 
+func validatePlatform(platform string) error {
```

Comment:
similarly can we move this to the `exec` pkg? it'd be awesome if we can move all this functionality to a completely separate `docker` or `docker/engine` pkg

Reply:
Yes! And I have a PR waiting in the wings (for after this one is merged) that splits `docker` into `dockerengine` and `dockerfile`.